docs/models/anthropic.md

```diff
@@ -78,6 +78,106 @@ agent = Agent(model)
 ...
 ```
 
+## Cloud Platform Integrations
+
+You can use Anthropic models through cloud platforms by passing a custom client to [`AnthropicProvider`][pydantic_ai.providers.anthropic.AnthropicProvider].
+
+### AWS Bedrock
+
+To use Claude models via [AWS Bedrock](https://aws.amazon.com/bedrock/claude/):
+
+=== "With Pydantic AI Gateway"
+
+    ```python {title="<a href='/gateway/' style='float: right;'>Learn about Gateway</a>" test="skip"}
+    from pydantic_ai import Agent
+
+    agent = Agent('gateway/bedrock:us.anthropic.claude-haiku-4-5-20251001-v1:0')
```

Comment:
Unlike the rest of this page, this will not actually use the `AnthropicModel` or the Anthropic-API(-running-on-Bedrock), but rather `BedrockConverseModel` and the AWS Bedrock Converse API (proxied via Gateway of course). Using the Anthropic model will pretty much always give better performance, so I'd prefer to not mention the gateway (or `BedrockConverseModel`) at all